addons/binding/org.openhab.binding.boschindego/deindego.map

```diff
@@ -0,0 +1,23 @@
+0=Status lesen
```

Comment:
This is exactly, what the state options are for in the Thing xml. If you define it there, the user does not have to do anything and he will see the (localized) texts for the error codes in the UI. You can put the english version in the xml and the german one in a property file in ESH-INF/i18n.